frontend/src/lib/lemon-ui/LemonToast/LemonToast.tsx

```diff
@@ -44,7 +44,7 @@ export interface ToastContentProps {
 export function ToastContent({ type, message, button, id }: ToastContentProps): JSX.Element {
     return (
         <div className="flex items-center" data-attr={`${type}-toast`}>
-            <span className="grow">{message}</span>
+            <span className="grow overflow-hidden text-ellipsis">{message}</span>
```

Comment:
text-ellipsis won't do anything without `whitespace-nowrap` right? (the `truncate` util contains all this but not sure if we actually want this as then it would only be one line...

Reply:
Hmm it adds the "..." at the end of the text.

With:
<img width="452" alt="Screenshot 2023-12-20 at 16 22 25" src="https://github.com/PostHog/posthog/assets/1851359/5789c87c-4107-4276-8387-d53918ee4572">

Without:
<img width="437" alt="Screenshot 2023-12-20 at 16 22 40" src="https://github.com/PostHog/posthog/assets/1851359/af9e1248-23d5-44c4-9d6b-85cc468beaae">

We do want more than one line :)